pkg/config/generate.go

```diff
@@ -74,6 +74,7 @@ func selectContainerImagesOrigin(cs *acsapi.OpenShiftManagedCluster) {
 		c.AzureCLIImage = "docker.io/microsoft/azure-cli:latest"
 
 		c.SyncImage = "quay.io/openshift-on-azure/sync:latest"
+		c.LogBridgeImage = "docker.io/jimminter/logbridge:latest"
```

Comment:
If we decide to go with this `logbridge` should we move this to a more official location?

Reply:
yes, I think I'll do that now.